src/Services/organizationService.js

```diff
@@ -0,0 +1,12 @@
+import { Get, Post } from "./privateApiService";
+
+const getOrganizationData = async () => {
+  const response = await Get("organization");
+  return response.data;
+};
+
+const updateOrganizationData = async (data) => {
+  await Post("organization", data);
```

Comment:
No faltaría in return acá?, tampoco se está manejando en caso que falle el await

Reply:
Solucionado